internals/cli/cmd_add.go

```diff
@@ -67,5 +67,11 @@ func (cmd *cmdAdd) Execute(args []string) error {
 }
 
 func init() {
-	addCommand("add", shortAddHelp, longAddHelp, func() flags.Commander { return &cmdAdd{} }, addDescs, nil)
+	AddCommand(&CmdInfo{
```

Comment:
These are a lot more readable on multiple lines and with the field names now, thanks!